migrator/clone/db_clone_manager_impl.go

```diff
@@ -128,18 +128,20 @@ func (d *dbCloneManagerImpl) Persist(cloneName string, pgClone string, persistBo
 			return err
 		}
 
+		// TODO (ROX-14524):  re-enable this in 3.75.  This will not occur until multiple upgrades of Postgres
```

Comment:
This will occur if you have multiple upgrades including 73 and 74 small upgrade. It will happen.

Reply:
Right.  any multiple upgrades.  Figured it is best to not do this now and save it for later, probably re-enable when the PVC is deprecated